### DISCUSSION TITLE
PretendardVariable.subset 폰트 질문드려요.

### DISCUSSION THREAD

**Discussion by Eunkyung-Son** [orioncactus/pretendard#169] [Q&A] [answered]
안녕하세요. 
https://github.com/orioncactus/pretendard/tree/main/packages/pretendard/dist/web/variable/woff2-dynamic-subset
위의 경로에서 
PretendardVariable.subset.0.woff2 부터 PretendardVariable.subset.91.woff2 까지의 폰트 파일이 분리 되어 있는데,
0부터 91까지의 숫자가 의미하는 바를 모르겠어요.
유추하기로는 1은 font weight 가 100, .. 9는 font weight가 900 을 의미 하는 것 같은데요 
문서를 읽어봐도 명확한 답을 찾을 수 없어서요.  도움 부탁드립니다.

**Comment by orioncactus** [comment, marked as answer]
굉장히 늦게 답장드립니다. 이제야 확인해 죄송합니다.

해당 디렉터리(woff-dynamic-subset 및 woff2-dynamic-subset)에서 제공하는 웹폰트는 다이나믹 서브셋으로, 폰트를 자주 쓰는 문자별로 그룹화해 92개 조각으로 나누고, 브라우저에 표시된 문자들만 선택적으로 불러올 수 있게 함으로써 웹폰트를 불러오는 용량을 유의미하게 절감할 수 있는 방식입니다. 따라서 0부터 91까지로 쓰인 숫자는 그룹별로 나뉘어진 순서를 의미합니다.

다이나믹 서브셋을 적용하는 방법응ㄴ 다음 URL에서 확인하실 수 있습니다: https://github.com/orioncactus/pretendard#다이나믹-서브셋
